config/generate_repository_config.yaml

```diff
@@ -5,23 +5,27 @@ mariadb:
   deb:
     path: https://dlm.mariadb.com/browse/mariadb_server/
     key: ["0x8167EE24", "0xE3C94F49", "0xcbcb082a1bb943db", "0xf1656f24c74cd1d8", "0x135659e928c12247"]
+  no_sublinks: true
 maxscale_ci:
   repo:
     path: https://mdbe-ci-repo.mariadb.net/Maxscale/
     old_key: https://mdbe-ci-repo.mariadb.net/MariaDBManager-GPG-KEY.public
     new_key: https://mdbe-ci-repo.mariadb.net/MDBE.public
+  no_sublinks: false
```

Comment:
Double negation is hard to read and reason.

Please switch from using Boolean values to usage of enumerations. I.e. user must declare a scan mode:
- follow-only-sublinks
- use-all-links

This data must be put inside the documentation for repository configuration.